pkgs/top-level/python-packages.nix

```diff
@@ -9648,6 +9648,10 @@ self: super: with self; {
 
   onnxruntime-tools = callPackage ../development/python-modules/onnxruntime-tools { };
 
+  onnxsim = callPackage ../development/python-modules/onnxsim { };
+
+  onnx-simplifier = callPackage ../development/python-modules/onnxsim { };
```

Comment:
this is because there are two pypi entries for this package ([onnxsim](https://pypi.org/project/onnxsim/) and [onnx-simplifier](https://pypi.org/project/onnx-simplifier/))

Reply:
Is there precedence for having these kinds of aliases (i.e. to reflect aliases in PyPI) in nixpkgs?

Reply:
yeah you are right. even onnx-simplifier package seems redundant.